src/utility/url.js

```diff
@@ -0,0 +1,36 @@
+
+export const MH_BACKEND_URL = process.env.REACT_APP_MH_BACKEND_URL
+export const CALENDAR_URL = process.env.REACT_APP_CALENDAR_API_URL + process.env.REACT_APP_CALENDAR_ID;
+export const ROSTER_URL = process.env.REACT_APP_ROSTER_API_URL + process.env.REACT_APP_ROSTER_ID;
```

Comment:
I don't think `url.js` should statefully know these URLs. each caller should maybe get these URLs from some `map.js` or `constants.js` file that pulls in all the envvars into `const` bois